apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -1181,6 +1289,31 @@ format_persistent_session_info(ClientId, PSInfo0) ->
     ),
     result_format_undefined_to_null(PSInfo).
 
+format_msg(#message{
+    id = ID,
+    qos = Qos,
+    topic = Topic,
+    from = From,
+    timestamp = Timestamp,
+    headers = Headers,
+    payload = Payload
+}) ->
+    Msg = #{
+        msgid => emqx_guid:to_hexstr(ID),
+        qos => Qos,
+        topic => Topic,
+        publish_at =>
+            emqx_utils_calendar:epoch_to_rfc3339(Timestamp),
```

Comment:
maybe should not format it.

Reply:
I followed retainer messages formatting...

Reply:
Removed time formatting